extensions/amp-story/1.0/_locales/af.js

```diff
@@ -20,7 +20,7 @@ import {LocalizedStringBundleDef, LocalizedStringId} from '../localization';
  * Localized string bundle used for af strings.
  * @const {!LocalizedStringBundleDef}
  */
-export default /** @const {!LocalizedStringBundleDef} */ ({
+const strings = {
```

Comment:
Are these changes required? Is there an impact on build size by creating a reference that's exported versus inlining?

Reply:
Yes, they're required. The typedefs in the current version of the code were added during a previous upgrade of closure compiler https://github.com/ampproject/amphtml/pull/18552/commits/f949f8f36f045314b270ffd85ad0f8d5735a35a7. It turns out that this isn't a valid way to annotate default exports (https://github.com/jsdoc3/jsdoc/issues/1132). Moreover, the code causes the latest version of the compiler to error out. 
```js
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.

CAST node should always have JSDocInfo
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.javascript.jscomp.TypeInference.traverse(TypeInference.java:671)
	at com.google.javascript.jscomp.TypeInference.traverseName(TypeInference.java:1269)
	at com.google.javascript.jscomp.TypeInference.traverseDeclarationChild(TypeInference.java:1192)
	at com.google.javascript.jscomp.TypeInference.traverseDeclaration(TypeInference.java:1184)
	at com.google.javascript.jscomp.TypeInference.traverse(TypeInference.java:656)
	at com.google.javascript.jscomp.TypeInference.flowThrough(TypeInference.java:344)
	at com.google.javascript.jscomp.TypeInference.branchedFlowThrough(TypeInference.java:356)
	at com.google.javascript.jscomp.TypeInference.branchedFlowThrough(TypeInference.java:76)
	at com.google.javascript.jscomp.DataFlowAnalysis$BranchedForwardDataFlowAnalysis.flow(DataFlowAnalysis.java:438)
	at com.google.javascript.jscomp.DataFlowAnalysis.analyze(DataFlowAnalysis.java:205)
	at com.google.javascript.jscomp.DataFlowAnalysis.analyze(DataFlowAnalysis.java:173)
	at com.google.javascript.jscomp.TypeInferencePass.inferScope(TypeInferencePass.java:121)
	at com.google.javascript.jscomp.TypeInferencePass$SecondScopeBuildingCallback.enterScope(TypeInferencePass.java:147)
	at com.google.javascript.jscomp.NodeTraversal.pushScope(NodeTraversal.java:1058)
	at com.google.javascript.jscomp.NodeTraversal.pushScope(NodeTraversal.java:1046)
	at com.google.javascript.jscomp.NodeTraversal.traverseWithScope(NodeTraversal.java:443)
	at com.google.javascript.jscomp.TypeInferencePass.inferAllScopes(TypeInferencePass.java:108)
	at com.google.javascript.jscomp.TypeInferencePass.process(TypeInferencePass.java:72)
	at com.google.javascript.jscomp.DefaultPassConfig$47$1.process(DefaultPassConfig.java:1906)
	at com.google.javascript.jscomp.PhaseOptimizer$NamedPass.process(PhaseOptimizer.java:328)
	at com.google.javascript.jscomp.PhaseOptimizer.process(PhaseOptimizer.java:237)
	at com.google.javascript.jscomp.Compiler.check(Compiler.java:1010)
	at com.google.javascript.jscomp.Compiler.performChecksAndTranspilation(Compiler.java:818)
	at com.google.javascript.jscomp.Compiler.lambda$stage1Passes$0(Compiler.java:748)
	at com.google.javascript.jscomp.CompilerExecutor$2.call(CompilerExecutor.java:102)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException: CAST node should always have JSDocInfo
```